src/modules/authentication/components/Registration/Registration.tsx

```diff
@@ -1,5 +1,231 @@
-function Registration() {
-	return <div></div>;
+import styles from './register.module.css'
+import logo from '../../../../assets/images/PMS 3.svg'
+import profileImg from '../../../../assets/images/profile.svg'
+import {useForm ,SubmitHandler }  from 'react-hook-form'
+import { toast } from "react-toastify";
+import { useState , useEffect} from 'react';
+import { useNavigate } from 'react-router-dom';
+import { AUTH_URLS, axiosInstance, BASE_AUTH} from '../../../../api';
+
+
+export default function Registration() {
+
+	const [isVisible, setVisible] = useState<boolean>(false);  
+	const [isRePasswordVisible, setIsRePasswordVisible] = useState<boolean>(false);  
+	const [isPasswordShown, setIsPasswordShown] = useState<boolean>(false)
+	const [isRePasswordShown, setIsRePasswordShown] = useState<boolean>(false)
+
+	const toggleHideShowPassword = ():void => {
+		setVisible(!isVisible);
+		
+		setIsPasswordShown(!isPasswordShown)
+		
+	  };
+	const toggleHideShowRePassword = ():void => {
+		setIsRePasswordVisible(!isRePasswordVisible);
+		
+		setIsRePasswordShown(!isRePasswordShown)
+		
+	  };
+
+
+const navigate = useNavigate()
+type FormFields = {
```

Comment:
I suggest moving the type outside of the component.

Reply:
done